cypress/integration/analytics.test.js

```diff
@@ -161,13 +172,12 @@ describe('select_item event', () => {
 
     let skuId
 
-    cy.getById('product-link')
-      .first()
-      .within(() => {
-        cy.getById('buy-button').then(($btn) => {
-          skuId = $btn.attr('data-sku')
-        })
+    cy.getById('store-card').first().click()
```

Comment:
Also here.

Reply:
I had to keep this one because it's part of the `select_item` event that is triggered when the product link is clicked and that's why we're starting from the PLP here and clicking the first product link. But I was able to simplify it as you recommended in [this other comment](https://github.com/vtex-sites/base.store/pull/131#discussion_r792863726).